modules/ROOT/partials/configuration/linkchecker_content_css.adoc

```diff
@@ -18,3 +18,16 @@ a[data-mce-linkchecker-focus="true"] {
 `+data-mce-linkchecker-focus+` attribute is set to _true_ every time a dedicated context menu is shown on a link. This only happens for `+invalid+` links.
 
 It is possible to replace or extend those styles, by providing a URL to custom stylesheet via `+linkchecker_content_css+` option.
+
+Type: `+String+`
+
+=== Example: Using `+linkchecker_content_css+`
+
+[source,js]
+----
+tinymce.init({
+    selector: 'textarea',
+    plugins: 'linkchecker',
+    linkchecker_content_css: 'http://mydomain.com/linkchecker_content.css'
+});
```

Comment:
This example has 4 spaces for indentation when it should be 2. It's also missing the comment after the selector.
```suggestion
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'linkchecker',
  linkchecker_content_css: 'http://mydomain.com/linkchecker_content.css'
});
```